### PR TITLE
Issue 1343 Fix: Hamcrest to Assertj

### PR DESCRIPTION
**Pull request by baruKreddy**
Signed-off-by: baruKreddy <bhargavirkotha@gmail.com>
## Type of change

 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java [hunk 1]
```diff
@@ -57,12 +57,9 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.hasSize;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.assertj.core.api.Assertions.*;
```
Comment (rohanKanojia) [inline jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java:62]: Please avoid using wildcard imports

File: jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java [hunk 2]
```diff
@@ -131,13 +128,10 @@ public void testGetTransitiveDependencies(@Mocked MavenProject mavenProject) {
...
     final List<Dependency> result = MavenUtil.getTransitiveDependencies(mavenProject);
     // Then
-    assertThat(result, hasSize(2));
-    assertThat(result, contains(
-        equalTo(Dependency.builder().groupId("org.eclipse.jkube").artifactId("foo-dependency").version("1.33.7")
-            .type("jar").scope("runtime").build()),
-        equalTo(Dependency.builder().groupId("org.eclipse.jkube").artifactId("bar-dependency").version("1.33.7")
-            .type("jar").scope("runtime").build())
-    ));
+      assertThat(result).hasSize(2);
```
Comment (rohanKanojia) [inline jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java:131]: Join these multiple assertions subject to one assertion chain.

```
assertThat(result)
  .hasSize(2)
  .contains(Dependency.builder().groupId("org.eclipse.jkube").artifactId("foo-dependency").version("1.33.7")
              .type("jar").scope("runtime").build(), Dependency.builder().groupId("org.eclipse.jkube").artifactId("bar-dependency").version("1.33.7")
              .type("jar").scope("runtime").build());
```

Comment (baruKreddy) [reply jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java:131]: Allow me to change according to suggested comments.

File: jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java (hunk 1 above)
Comment (baruKreddy) [reply jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java:62]: Sure @rohanKanojia , i will change it

File: jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java [hunk 3]
```diff
@@ -57,110 +57,103 @@
...
 
 public class MavenUtilTest {
-  @Rule
-  public TemporaryFolder temporaryFolder = new TemporaryFolder();
-  @Mocked
-  private KitLogger log;
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+    @Mocked
+    private KitLogger log;
```
Comment (rohanKanojia) [inline jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java:68]: You seem to have changed indentation now to 4 characters. Please configure your IDE as per project requirements https://www.eclipse.org/jkube/contributing#ide-config-and-code-style

Comment (baruKreddy) [reply jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java]: Thanks for the resource, will make sure to format in next merge as well. Resolved.

**Review by rohanKanojia — APPROVED** [review]

**Review by manusa — APPROVED** [review]
LGTM, thx!
Please note that in general we prefer you don't format the complete file, but only the changes you made.
Else, it makes it really hard to review.

**Pull request merged**